### PR TITLE
Latest toolchain SDK produced from otc-api swagger

### PR DESCRIPTION
**Pull request by gmarjoram**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [x] Yes
- [ ] No

### CODE REVIEW

File: cdtoolchainv2/cd_toolchain_v2.go
Comment (briangleeson) [inline cdtoolchainv2/cd_toolchain_v2.go:39]: This URL looks wrong, it should be
```suggestion
// <a href="https://cloud.ibm.com/apidocs/iam-identity-token-api">Identity and Access Management (IAM)
```

**Review by briangleeson — CHANGES_REQUESTED** [review]

File: cdtoolchainv2/cd_toolchain_v2.go
Comment (gmarjoram) [reply cdtoolchainv2/cd_toolchain_v2.go:39]: @omar-albastami this needs to be fixed in the `otc-api` swagger

Comment (gmarjoram) [reply cdtoolchainv2/cd_toolchain_v2.go]: Should be fixed @briangleeson

**Review by briangleeson — APPROVED** [review]
lgtm

As discussed let's hold off merging until the other pull requests are merged

**Pull request merged**
